stl/src/winapinls.cpp

```diff
@@ -5,10 +5,495 @@
 
 #if _STL_WIN32_WINNT < _WIN32_WINNT_VISTA
 
-#include "nlsdownlevel.h"
 #include <ctype.h>
 #include <stdlib.h>
 
+namespace {
+struct LCIDTOLOCALENAME {
+   LCID lcid;
+   const wchar_t* localeName;
+};
+
+

```

Comment:
One blank line is plenty between these declarations. (Yes, I know there were 2 in the existing file, this is "new conventions vs. old conventions".)
```suggestion
```